### PR TITLE
Moving disclaimer comments inline

### PR DESCRIPTION
**Pull request by kylestarr**
Moving disclaimer comments inline to better specify which boolean statement is being referred to.

### CODE REVIEW

File: getting_started/first_2d_game/03.coding_the_player.rst [hunk 1]
```diff
@@ -275,8 +275,7 @@ movement. Let's place this code at the end of the ``_process()`` function:
         if velocity.x != 0:
             $AnimatedSprite2D.animation = "walk"
             $AnimatedSprite2D.flip_v = false
-            # See the note below about boolean assignment.
-            $AnimatedSprite2D.flip_h = velocity.x < 0
+            $AnimatedSprite2D.flip_h = velocity.x < 0 ## See the note below about this boolean assignment.
```
Comment (mhilbrunner) [inline getting_started/first_2d_game/03.coding_the_player.rst:278]:
```suggestion
            $AnimatedSprite2D.flip_h = velocity.x < 0 # See the note below about this boolean assignment.
```

**Comment by YuriSizov** [comment]
I don't think this helps to better specify. It doesn't change which line the comment refers to, it just makes the snippet much wider, which is a problem for codeblocks in samples. So IMO this just makes it harder to read the docs.

**Comment by kylestarr** [comment]
> I don't think this helps to better specify. It doesn't change which line the comment refers to, it just makes the snippet much wider, which is a problem for codeblocks in samples. So IMO this just makes it harder to read the docs.

@YuriSizov That is fair. And I had considered that when creating this PR. Perhaps the better direction is to simply add to the comment that it's referring to the boolean statement below rather than the one prior. See new commit https://github.com/godotengine/godot-docs/pull/8832/commits/af66da052fb276ad2d5182888d71fa99515abc3b

**Review by skyace65 — APPROVED** [review]

File: getting_started/first_2d_game/03.coding_the_player.rst [hunk 2]
```diff
@@ -275,7 +275,7 @@ movement. Let's place this code at the end of the ``_process()`` function:
         if velocity.x != 0:
             $AnimatedSprite2D.animation = "walk"
             $AnimatedSprite2D.flip_v = false
-            # See the note below about boolean assignment.
+            ## See the note below about the following boolean assignment.
```
Comment (skyace65) [inline getting_started/first_2d_game/03.coding_the_player.rst:278]:
```suggestion
            # See the note below about the following boolean assignment.
```

**Review by skyace65 — CHANGES_REQUESTED** [review]

**Review by skyace65 — APPROVED** [review]

**Comment by skyace65** [comment]
Thanks!

**Comment by mhilbrunner** [comment]
Cherry-picked to 4.2 in https://github.com/godotengine/godot-docs/pull/9647.

**Pull request merged**